### DISCUSSION TITLE
[Feature] one-click installation of Postiz App

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [coollabsio/coolify#3372] [Feature Requests]
This open source social media scheduling tool is open-source and offers a self-hosting option. Would love to see it included in the services list
https://github.com/gitroomhq/postiz-app

**Comment by dr-diffie** [comment]
They even have a full Documentation for it. 

https://docs.postiz.com/installation/coolify

**Reply by TheMikeyRoss** [reply]
True, but a one-click installer within Coolify would be great

**Comment by samirariss** [comment]
Yeah, It's super useful and has an active community

**Comment by tidewatch** [comment]
Would be great to have on Coolify

**Reply by samirariss** [reply]
Already available. Just pick Postiz when choosing a service.
